bin/generate_analytics_JSONL_files.sh

```diff
@@ -25,21 +25,21 @@ SOLR_HOST=$(get_host_from_hostport $SOLR_HOST)
 
 require_env_var "SOLR_PORT"
 
-JAVA_OPTS="-Dsolr.host=$SOLR_HOST"
-JAVA_OPTS="$JAVA_OPTS -Dsolr.port=$SOLR_PORT"
-JAVA_OPTS="$JAVA_OPTS -Dzk.host=$ZK_HOST"
-JAVA_OPTS="$JAVA_OPTS -Dzk.port=$ZK_PORT"
-JAVA_OPTS="$JAVA_OPTS -Ddata.files.location=$BIOENTITIES"
-JAVA_OPTS="$JAVA_OPTS -Dexperiment.files.location=$EXPERIMENT_FILES"
-JAVA_OPTS="$JAVA_OPTS -Djdbc.url=$jdbc_url"
-JAVA_OPTS="$JAVA_OPTS -Djdbc.username=$jdbc_username"
-JAVA_OPTS="$JAVA_OPTS -Djdbc.password=$jdbc_password"
-JAVA_OPTS="$JAVA_OPTS -Dserver.port=$server_port"
```

Comment:
I changed this back to underscore because I would like to be able to override the expected `$JAVA_OPTS` from outside, to set things like heap size or others.